test/bake/dev/server-sourcemap.test.ts

```
@@ -0,0 +1,149 @@
+function helperFunction() {
+  throw new Error("Nested error");
+}`,
+  },
+  framework: "react",
+  async test(dev) {
+    await Promise.all([dev.fetch("/nested").catch(() => {}), dev.output.waitForLine(/Nested error/)]);
+
+    // Check that stack trace shows both files with correct lines
+    const lines = dev.output.lines.join("\n");
+    // Strip ANSI codes for cleaner checking
+    const cleanLines = lines.replace(/\x1b\[[0-9;]*m/g, "");
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)